### PR TITLE
chore: Add make command to init db from dev server (DEV-1989)

### PR DESCRIPTION
**Pull request by EricSommerhalder**
### Task Description/Number

Issue Number: DEV-1989

### Basic Requirements

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### PR Type

What kind of change does this PR introduce?

- [ ] fix: represents bug fixes
- [ ] refactor: represents production code refactoring
- [ ] feat: represents a new feature
- [ ] docs: documentation changes (no production code change)
- [ ] chore: maintenance tasks (no production code change)
- [ ] test: all about tests: adding, refactoring tests (no production code change)
- [ ] other... Please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No
- [ ] Maybe (not 100% sure => check with FE)

### Does this PR change client-test-data?

- [ ] Yes (don't forget to update the JS-LIB team about the change)
- [ ] No

### CODE REVIEW

**Review by BalduinLandolt — APPROVED** [review]
LGTM

**Review by irinaschubert — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -289,6 +289,11 @@ init-db-test-from-prod: db_prod_dump.trig init-db-test-empty ## init local datab
 	@curl -X POST -H "Content-Type: application/sparql-update" -d "DROP ALL" -u "admin:test" "http://localhost:3030/knora-test"
 	@curl -X POST -H "Content-Type: application/trig" -T "${CURRENT_DIR}/db_prod_dump.trig" -u "admin:test" "http://localhost:3030/knora-test"
 
+.PHONY: init-db-test-from-dev
+init-db-test-from-dev: ## init local database with data from staging. Use as `make init-db-test-from-dev PW=database-password`
+	@echo $@
+	${MAKE} init-db-from-env PW=${PW} ENV=db.dev.dasch.swiss
```
Comment (BalduinLandolt) [inline Makefile:295]: I wonder, does it actually need the `PW=${PW}` or would that propagate through anyways?

**Review by BalduinLandolt — APPROVED** [review]
noted just a thought

File: Makefile [hunk 2]
```diff
@@ -289,6 +289,11 @@ init-db-test-from-prod: db_prod_dump.trig init-db-test-empty ## init local datab
 	@curl -X POST -H "Content-Type: application/sparql-update" -d "DROP ALL" -u "admin:test" "http://localhost:3030/knora-test"
 	@curl -X POST -H "Content-Type: application/trig" -T "${CURRENT_DIR}/db_prod_dump.trig" -u "admin:test" "http://localhost:3030/knora-test"
 
+.PHONY: init-db-test-from-dev
+init-db-test-from-dev: ## init local database with data from staging. Use as `make init-db-test-from-dev PW=database-password`
```
Comment (mpro7) [inline Makefile:293]: Yous should be more clear in documentation here. For now DEV = STAGING, but in the near future most probably this will change.

**Pull request merged**